src/index.css

```diff
@@ -28,15 +28,15 @@
   }
 
   .yellow-button {
-    @apply font-titilliumSemiBold rounded py-[13px] px-[25px] shadow-sm bg-yellow-500 sm:w-auto h-fit text-center lg:w-48 w-32 lg:text-base text-xs;
+    @apply font-titilliumSemiBold rounded sm:py-[13px] py-[8px] sm:px-[25px] px-[15px] shadow-sm bg-yellow-500 sm:w-auto h-fit text-center lg:w-48 w-32 sm:text-xl text-sm min-w-fit;
   }
 
   .title {
-    @apply font-titilliumSemiBold sm:text-3xl lg:text-4xl text-2xl;
+    @apply font-titilliumBold sm:text-3xl lg:text-4xl text-2xl;
   }
 
   .body {
```

Comment:
Nu cred ca o clasa numita "body" e ok dpdv semantica. 
Nu a fost adaugata in PR-ul asta dar ar trebui modificata clasa. 

Daca intentia e sa fie folosita ca "text de sub titlu" ar putea sa se numeasca article, sau article-font.

Reply:
body este pentru contentul din landing si about, article o sa fie adaugat pentru restul de texte gri din website (pe pagina de ong-uri si cardul de organizatie / detalii program de practica)

Reply:
Si este intuitiv body ca si clasa pentru texte? Voiam sa zic ca ar trebui redenumit.